frontend/lib/types/CompositeType.cpp

```diff
@@ -183,7 +183,8 @@ bool CompositeType::isMissingBundledClassType(Context* context, ID id) {
   if (noLibrary) {
     auto path = id.symbolPath();
     return path == "ChapelReduce.ReduceScanOp" ||
-           path == "Errors.Error";
+           path == "Errors.Error" || 
+           path == "CTypes.c_ptr";
```

Comment:
I'm interested in when you hit this, because it seems to suggest that a `CompositeType` is being created from `CTypes.c_ptr`. I believe we do not want this to happen, because the composite `CTypes.c_ptr` type will not behave the same as the builtin `CPtrType`. So rather than allowing it to pass in these checks, it might be worth investigating and patching the code that creates the composite version.

Reply:
And to be clear, I wouldn't hold the PR over that if you want to get it in for feature freeze. Feel free to open an issue and note that as future work on this PR.

Reply:
This came up when generating the `asttag` based on the id. I'll open a new issue to discuss what we want to intended behavior to be.